ard/cost/orbit_wrap.py

```diff
@@ -0,0 +1,788 @@
+
+    def setup(self):
+        """Define all input variables from all models."""
+
+        # load modeling options
+        self.modeling_options = self.options["modeling_options"]
+        self.N_turbines = self.modeling_options["farm"]["N_turbines"]
+        self.N_substations = self.modeling_options["farm"]["N_substations"]
+
+        self.set_input_defaults("wtiv", "example_wtiv")
+        self.set_input_defaults("feeder", "example_feeder")
+        # self.set_input_defaults("num_feeders", 1)
+        # self.set_input_defaults("num_towing", 1)
+        # self.set_input_defaults("num_station_keeping", 3)
+        # self.set_input_defaults(
+        #    "oss_install_vessel", "example_heavy_lift_vessel",
+        # )
+        self.set_input_defaults("site_distance", 40.0, units="km")
+        self.set_input_defaults("site_distance_to_landfall", 40.0, units="km")
+        self.set_input_defaults("interconnection_distance", 40.0, units="km")
+        self.set_input_defaults("plant_turbine_spacing", 7)
+        self.set_input_defaults("plant_row_spacing", 7)
+        self.set_input_defaults("plant_substation_distance", 1, units="km")
+        # self.set_input_defaults("num_port_cranes", 1)
+        # self.set_input_defaults("num_assembly_lines", 1)
+        self.set_input_defaults("takt_time", 170.0, units="h")
+        self.set_input_defaults("port_cost_per_month", 2e6, units="USD/mo")
+        self.set_input_defaults("construction_insurance", 44.0, units="USD/kW")
+        self.set_input_defaults("construction_financing", 183.0, units="USD/kW")
+        self.set_input_defaults("contingency", 316.0, units="USD/kW")
+        self.set_input_defaults("commissioning_cost_kW", 44.0, units="USD/kW")
+        self.set_input_defaults("decommissioning_cost_kW", 58.0, units="USD/kW")
+        self.set_input_defaults("site_auction_price", 100e6, units="USD")
+        self.set_input_defaults("site_assessment_cost", 50e6, units="USD")
+        self.set_input_defaults("construction_plan_cost", 1e6, units="USD")
+        self.set_input_defaults("installation_plan_cost", 2.5e5, units="USD")
+        self.set_input_defaults("boem_review_cost", 0.0, units="USD")
+
+        self.add_subsystem(
+            "orbit",
+            ORBITWisdemDetail(
+                modeling_options=self.modeling_options,
+                case_title=self.options["case_title"],
+                floating=self.options["floating"],
+                jacket=self.options["jacket"],
+                jacket_legs=self.options["jacket_legs"],
+            ),
+            promotes=["*"],
+        )
+
+
+class ORBITWisdemDetail(orbit_wisdem.OrbitWisdem):
+    """ORBIT-WISDEM Fixed Substructure API, modified for detailed layouts"""
+
+    _path_library = None
+
+    def initialize(self):
+        super().initialize()
+
+        self.options.declare("case_title", default="working")
+        self.options.declare("modeling_options")
+
+    def setup(self):
+        """Define all the inputs."""
```

Comment:
I'm not sure we need to exhaustively define all the ORBIT input here. It seems to me that all the discrete inputs here could be set via modeling options. Defining them all here duplicates what is already done in WISDEM and seems like unnecessary code for Ard to own and maintain.